yarn-project/circuits.js/src/cbind/circuits.gen.ts

```diff
@@ -0,0 +1,954 @@
+/* eslint-disable */
+// GENERATED FILE DO NOT EDIT
```

Comment:
Including in git so people only need to call remake-bindings if something changes